instance/templates/instance/memcached/memcached.nomad

```diff
@@ -0,0 +1,53 @@
+# Jobs need a globally consistent name, so we need to include the instance id.
+job "memcached-1234" {
+  datacenters = ["dc1"]
+  type = "service"
+  constraint {
+    distinct_hosts = true
+  }
+  update {
+    max_parallel = 1
+  }
+  migrate {
+    max_parallel = 1
+  }
+  group "memcached" {
+    count = 3    # Only 1 for stage
+    task "memcached" {
+      driver = "exec"
+      config {
+        command = "memcached"
```

Comment:
Apparently the docs say this value should be absolute:

> If executing a binary that exists on the host, the path must be absolute.

But somehow this worked? (I don't know why it has to be absolute though TBH, that isn't explained).

Reply:
Obviously, this _isn't_ required to be an absolute path, but if that's what's documented, I'll change it.  (I've force-pushed the change over the old commit, so you can't see the diff, but it's the only thing I changed.)

Reply:
Hmm, for some reason I don't actually see any difference though anyway o_0

Reply:
My bad, I changed the semantics of one of my git aliases before my vacation, and forgot about the change in the last few weeks.  Now it's _actually_ updated.